lib/xcpretty/formatters/formatter.rb

```diff
@@ -114,11 +116,20 @@ def format_error(message)
       "\n#{red(error_symbol + " " + message)}\n\n"
     end
 
+    def format_phase_script_error(error, output)
+      indentedOutput = output.join("    ");
```

Comment:
Do not use semicolons to terminate expressions.<br>Use snake_case for variable names.